### DISCUSSION TITLE
欢迎来到本项目的讨论组

### DISCUSSION THREAD

**Discussion by dirk1983** [dirk1983/chatgpt#19] [Announcements]
## 👋 欢迎各位！
  我们正在使用讨论区与社区中的其他成员建立联系。我们希望您：
  * 提出您的建议
  * 分享您的想法
  * 多参与问题和互动
  * 保持开放的心跳

有你更精彩！ 💪

**Comment by superw2018** [comment]
我感觉这个流不是真正的实时返回，页是接口把所有回答都调用好才输出的，所以感觉输出有点慢，一直说在组织语言。

**Comment by chaoxinghou** [comment]
感谢开源，自己用了这个源码进行微调搭了一个服务，对接了数据库，欢迎指教。快速体验：houcx.com
